### PR TITLE
docs(README): Add documentation for rpm packages.

### PR DESCRIPTION
**Pull request by Dark-Passenger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,37 @@ sudo apt-get remove etcher-electron
 sudo rm /etc/apt/sources.list.d/etcher.list
 sudo apt-get update
 ```
+#### Redhat(RHEL) and Fedora based Package Repository (GNU/Linux x86/x64)
```
Comment (jviotti) [inline README.md]: Should we add a space between `Redhat` and `(RHEL)`, for consistency purposes?

File: README.md [hunk 2]
```diff
@@ -67,6 +67,37 @@ sudo apt-get remove etcher-electron
...
+    ```sh
+    sudo dnf install -y etcher-electron
+    ```
+
+##### Uninstall
+
+```
+sudo yum remove -y etcher-electron
+sudo rm /etc/yum.repos.d/bintray-resin-io-redhat.repo
+sudo yum clean all; sudo yum makecache fast
```
Comment (jviotti) [inline README.md]: Should we split these commands into two lines?

**Comment by jviotti** [comment]
Looks great, thanks a lot @Dark-Passenger !

**Comment by lurch** [comment]
Indeed, thanks! :smiley:

**Pull request merged**